react-multiple-components/my-app/src/PrevNextButtons.tsx

```diff
@@ -0,0 +1,8 @@
+type Props = {
+  text: string;
+  onClickHandler?: () => void;
```

Comment:
`onClick` is a better name than `onClickHandler`. The name should be like `onEventName`. The parent provides the handler, so putting it in the prop name is odd.